reference/render/viewer.html

```diff
@@ -0,0 +1,96 @@
+<header>
```

Comment:
Dit is een reference file `reference/render/view.html`, maar waar is de handlebars variant? Hetzelfde geldt voor viewer2.html..

Changes/creations in .html files kunnen alleen komen door hun .hbs of .yml counterpart

Reply:
Het lijkt er zelfs op alsof de ingecheckte reference files niet bij jouw changes hoort. Zie de laatste build: https://travis-ci.com/dso-toolkit/dso-toolkit/builds/125388199#L314

Deze foutmelding geeft aan dat er een verschil zit tussen de referentiefiles (de desired markup) en wanneer de build server de .hbs+.yml omzet naar .html files

Reply:
Deze file zou er inderdaad compleet niet in moeten zitten is het genoeg als ik deze files simpelweg verwijder ?

Reply:
Dat kan je proberen, maar het is pas goed als de build slaagt